integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -228,7 +229,13 @@ public int getattr(String path, FileStat stat) {
   private int getattrInternal(String path, FileStat stat) {
     final AlluxioURI uri = mPathResolverCache.getUnchecked(path);
     try {
-      URIStatus status = mFileSystem.getStatus(uri);
+      URIStatus status = null;
+      // Handle special metadata cache operation
+      if (mConf.getBoolean(PropertyKey.FUSE_SPECIAL_COMMAND_ENABLED)
+          && mFuseCLientTools.isReservedPath(uri)) {
```

Comment:
If the AlluxioFuse mount point mapping to Alluxio no-root path, the `uri` should always starts with the mounted Alluxio path, so you should remove prefix of uri when call `isReservedPath` method

Reply:
@maobaolong The reserved dir is not only be identified by fuse root /.alluxiocli, but for any path under the the fuse mount point works, for example:
If mount point  mapping to Alluxio no-root path /dir, the uri above will be /dir/.alluxiocli, it is considered to be a reserved path too, so the follow procedure will be executed normally.

Reply:
@bzheng888 OK